dds_web/templates/mail/project_release.html

```diff
@@ -18,22 +18,28 @@
         </ul>
       </p>
       <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">
+        You were added to this project on behalf of {{displayed_sender}}.
+      </p>
+      <p style="font-family: arial, sans-serif; font-size: 12px; font-weight: normal; margin: 0; Margin-top: 0px;">
+        To list the files in this project, run: <br>
+        <i><mark id="ProjectListing">dds ls -p {{project_id}}</mark></i>. <button onclick="copyTextProjectRelease(1)">Copy command</button></p>
+      <p style="font-family: arial, sans-serif; font-size: 12px; font-weight: normal; margin: 0; Margin-top: 0px;">
+        To download all the files in this project to your current directory, run: <br>
+        <i><mark id="ProjectDownload">dds data get -p {{project_id}} -a --verify-checksum</mark></i>. <button onclick="copyTextProjectRelease(2)">Copy command</button> </p>
+      <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-top: 15px;">
+        For more information (including an installation guide), see the DDS CLI documentation here: <a href="https://scilifelabdatacentre.github.io/dds_cli/">scilifelabdatacentre.github.io/dds_cli</a>. </p>
+      <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-top: 15px;">
+        
         {% if unit_email %}
-        You were added to this project on behalf of {{displayed_sender}} (<a href="mailto: {{unit_email}}">{{unit_email}}</a>).
+        if you experience issues, please contact the SciLifeLab unit <b>{{displayed_sender}}</b> at (<a href="mailto: {{unit_email}}">{{unit_email}}</a>).
```

Comment:
Remove the parenthesis
```suggestion
        if you experience issues, please contact the SciLifeLab unit <b>{{displayed_sender}}</b> at <a href="mailto: {{unit_email}}">{{unit_email}}</a>.
```